tests/unit/NumberUtilsTest.js

```diff
@@ -0,0 +1,14 @@
+const NumberUtils = require('../../src/libs/NumberUtils');
+
+const result = NumberUtils.rand64();
+
+const max64 = 9223372036854775807;
+
+describe('libs/NumberUtils', () => {
+    it('should generate a random 64-bit numeric string', () => {
+        const id = NumberUtils.rand64();
+        expect(isNaN(result)).toBe(false);
```

Comment:
I think this is what you are looking for
```suggestion
        expect(isNaN(id));
```